### PR TITLE
Add Peer Support to substance use list

### PR DESCRIPTION
**Pull request by jessethach**
## Changes

- Update translations
- Map to key of translations
- Adjust unit test

## Screenshots

Screen Shot 2022-11-17 at 10 16 44 AM

## Checklist

- [ ] if adds a new page, adds accessibility tests
- [ ] if adds a new page or new interaction, adds e2e test
- [ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc](https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)
- [ ] if changes filter functionality, updates [filters doc](https://docs.google.com/document/d/1yEdo7IpHCtEKNNF6FMLapBUgcPw_8CY7wGwxKTdtJrU/edit)

https://app.asana.com/0/1202690670605811/1202843153961770

### CODE REVIEW

File: src/translations/en_translations.json [hunk 1]
```diff
@@ -227,7 +228,7 @@
   "languageShortValuesVietnamese": "Tiếng Việt",
   "feesTitle": "Payment types accepted",
   "feesValuesSelfPay": "Self pay",
-  "feesValuesMedicaid": "Medicaid",
+  "feesValuesMedicaid": "Health First Colorado (Colorado's Medicaid Program)",
```
Comment (jlhogan) [inline src/translations/en_translations.json:231]: we have been manually un-doing this change until confirmation from BHA that this language is what we want to move forward with. Ideally, changes shouldn't go into the google sheet until they are finalized but alas...

File: src/types/care_provider.ts [hunk 2]
```diff
@@ -123,7 +123,7 @@ export type CareProvider = {
   substanceUse: {
     supported: boolean;
     duiSupported: boolean;
-    services: { [key in SubstanceUseServices]: boolean };
+    services: { [key in SubstanceUseServices]: boolean } & { PeerSupport: boolean };
```
Comment (jlhogan) [inline src/types/care_provider.ts:126]: can we add a comment somewhere (here, or in the data parsing script, or both) to just explain in plain language why we're doing this (this = augmenting the substance use services with this separate field) so future developers will understand that this is intentional and a result to transform the data from it's original structure to better contextualize it for the end user

**Review by jlhogan — APPROVED** [review]

File: src/translations/en_translations.json (hunk 1 above)
Comment (jessethach) [reply src/translations/en_translations.json:231]: Good to know! Will revert.

File: src/types/care_provider.ts (hunk 2 above)
Comment (jessethach) [reply src/types/care_provider.ts:126]: Will do! I'll write the comment here.

File: src/data/process_data.ts [hunk 3]
```diff
@@ -138,7 +140,7 @@ const transformRow = (row: InputRow): CareProvider => {
         row.OpioidTreatmentPrograms === "1"
       ),
       duiSupported: row.DUIDWI === "1",
-      services: getBooleanMap(SUBSTANCE_USE_SERVICES, substanceUseServices),
+      services: { PeerSupport: !!row.RSSOServicesProvided, ...getBooleanMap(SUBSTANCE_USE_SERVICES, substanceUseServices) },
```
[the remark this answers is not in the document]
Comment (jessethach) [reply src/data/process_data.ts:143]: Good question. I'll leave the question in the ticket for follow up with BHA.

File: src/translations/en_translations.json (hunk 1 above)
Comment (jessethach) [reply src/translations/en_translations.json]: This is resolved.

File: src/types/care_provider.ts (hunk 2 above)
Comment (jessethach) [reply src/types/care_provider.ts:133]: Added comment.

File: src/data/process_data.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (jessethach) [reply src/data/process_data.ts:143]: Question posted https://app.asana.com/0/0/1202843153961770/1203396923038530/f

[the remark this answers is not in the document]
Comment (jessethach) [reply src/data/process_data.ts]: Question answered by Vicki:

> You can ignore RSSOServicesProvided — down the line (once they start issuing RSSO license) that will be the field that stores the various types of peer support a provider offers. ActiveRSSOLicense is the flag that we want to use to determine if the "Peer Support" bullet should appear or not.
> 
> Let me know if that does/doesn't make sense or if you have any additional questions.

https://app.asana.com/0/0/1202843153961770/1203402795271105/f

I removed the `RSSOServicesProvided` prop from the Input type and set the `PeerSupport` prop from the `ActiveRSSOLicense` column. This should now be resolved.

**Pull request merged**